src/types/component.rs

```diff
@@ -47,11 +47,44 @@ pub struct InputStream {
 pub struct View {
     pub id: Option<ComponentId>,
     pub children: Option<Vec<Component>>,
+
+    /// Width of a component in pixels. Required when using absolute positioning.
     pub width: Option<usize>,
+    /// Height of a component in pixels. Required when using absolute positioning.
     pub height: Option<usize>,
+
+    /// Direction defines how static children are positioned inside the View.
+    /// "row" - Children positioned from left to right.
+    /// "column" - Children positioned from top to bottom.
+    pub direction: Option<ViewDirection>,
+
+    /// Distance between the top edge of this component and the top edge of its parent.
+    /// If this field is defined, then component will ignore a layout defined by its parent.
```

Comment:
I don't think this line is super clear. How about `If this field is defined, this element will be absolutely positioned, instead of being laid out by it's parent`?